WooCommerce/Classes/ViewRelated/Feature Highlight/TooltipPresenter.swift

```diff
@@ -163,12 +168,12 @@ final class TooltipPresenter {
             case .top:
                 tooltipTopConstraint = tooltip.topAnchor.constraint(
                     equalTo: containerView.topAnchor,
-                    constant: targetPoint().y)
+                    constant: targetPoint().y + totalVerticalBuffer)
             }
         }
 
-        tooltipConstraints.append(tooltipTopConstraint!)
-        NSLayoutConstraint.activate(tooltipConstraints)
+        tooltipConstraints.append(tooltipTopConstraint)
+        NSLayoutConstraint.activate(tooltipConstraints.compactMap { $0 })
```

Comment:
Nit: to avoid having to keep the array item optional, we can create a local variable of `tooltipConstraints`, add it to the array and update the global variable to reference it.